source_code/src/FLASH/flash_mem.c

```diff
@@ -42,7 +42,9 @@
 void memoryBoundaryErrorCallback(void)
 {
     // We'll add more debug later if needed
+#ifndef MINI_BOOTLOADER
     usbPutstr("#MBE");
+#endif
```

Comment:
Oh I see. I just had this error before, but now its gone. Seems you are right. Only the definition in the makefile was missing. And the commit I cherry picked added both. Thanks.